common/SigUtil.cpp

```diff
@@ -295,6 +295,8 @@ namespace SigUtil
     static
     void handleTerminationSignal(const int signal)
     {
+        const auto onrre = errno; // Save.
```

Comment:
Nice, joke, but perhaps go with something like oldErrno or origErrno to be more readable?